chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -579,12 +596,12 @@
 
   node.set[:provisioner][:available_oses][os] ||= Mash.new
   if /^(hyperv|windows)/ =~ os
-    node.set[:provisioner][:available_oses][os][:kernel] = "../#{os}/#{kernel}"
+    node.set[:provisioner][:available_oses][os][:kernel] = "../../#{os}/#{kernel}"
     node.set[:provisioner][:available_oses][os][:initrd] = " "
     node.set[:provisioner][:available_oses][os][:append_line] = " "
   else
-    node.set[:provisioner][:available_oses][os][:kernel] = "../#{os}/install/#{kernel}"
-    node.set[:provisioner][:available_oses][os][:initrd] = "../#{os}/install/#{initrd}"
+    node.set[:provisioner][:available_oses][os][:kernel] = "../../#{os}/install/#{kernel}"
```

Comment:
Looks like that `#{kernel}` and `#{initrd}` is going to be replaced with `../something`, isn't better to avoid this `install` part of the path directly?